src/js/util.js

```diff
@@ -62,7 +63,7 @@ function findSingleNodeValue(xpath, elem) {
             null
         ).singleNodeValue;
     } catch (ex) {
-        console.warn('findSingleNodeValue blew up with: ' + xpath);
+        console.log('findSingleNodeValue blew up with: ', xpath);
```

Comment:
top banana!
I think "blew up" is perhaps hyperbole - we really only need to log when we fall off the end of the list of parsing options. (see my comment about nested exceptions above).

Reply:
Oddly, it looks like lower-case() causes it to blow up, and apparently isn't needed anyway; it seems to compare case-insensitive already.

Reply:
could lowercase be blowing up on a null/undefined?
LGTM btw